R/ui-legacy.R

```diff
@@ -137,24 +148,20 @@ ui_warn <- function(x, .envir = parent.frame()) {
 }
 
 
-# Silence -----------------------------------------------------------------
-
-#' @rdname ui
-#' @param code Code to execute with usual UI output silenced.
-#' @export
-ui_silence <- function(code) {
-  withr::with_options(list(usethis.quiet = TRUE), code)
-}
 
 # Questions ---------------------------------------------------------------
-
 #' User interface - Questions
 #'
-#' These functions are used to interact with the user by posing a simple yes or
-#' no question. For details on the other `ui_*()` functions, see the [ui] help
-#' page.
+#' @description
+#' `r lifecycle::badge("superseded")`
+#'
+
+#' `ui_yeah()` and `ui_nope()` are technically superseded, but, unlike the rest
+#' of the legacy [`ui_*()`][ui-legacy-functions] functions, there's not yet a
+#' drop-in replacement available in the [cli package](https://cli.r-lib.org/).
+#' `ui_yeah()` and `ui_nope()` are no longer used internally in usethis.
```

Comment:
`ui_yeah()` and `ui_nope()` are awkward. I have marked them as superseded but there's not really a cli alternative yet.

Internally usethis uses new functions `ui_yep()` and `ui_nah()`. These are still just wrappers around `utils::menu()`, so the main difference is that I'm using cli to prepare the `title` and `choices`.